### DISCUSSION TITLE
Number of reads mapped to a contig

### DISCUSSION THREAD

**Discussion by JLammons14** [metagenome-atlas/atlas#599] [Q&A] [answered]
Hello Silas, 

I had a quick question for you, sorry if there is already an answer here, but I having been using the contigs generated from ATLAS to identify bacteriophage in my samples. I wanted to know if there is an output in atlas that would tell me how many reads mapped to the bacteriophage contig or some similar metric of abundance. 

I saw this post: https://github.com/metagenome-atlas/atlas/issues/186 ,  regarding a somewhat similar problem but I dont think it quite addresses what I was looking for, so thought I would ask this here. 

Thanks for your time, sorry if this info was provided somewhere else.

**Comment by SilasK** [comment, marked as answer]
I think you need to look at the coverage statistics for each file "{sample}/assembly/contig_stats/postfilter_coverage_stats.txt"`

there is also a binned version of this file. e.g. coverage information for each 1kb window. 

I agree this could be better described.
